spec/models/slide_spec.rb

```diff
@@ -37,16 +37,59 @@
     end
   end
 
-  describe 'thumbnail_url' do
-    it 'url is valid' do
+  describe 'Creating "Slide" model' do
+    success_data = {user_id: 1, name: 'dummy', description: 'dummy', object_key: 'dummy', category_id: 1}
+    it "is valid with user_id, name, description, object_key and category" do
+      slide = Slide.new(success_data)
+      expect(slide.valid?).to eq(true)
+    end
+
+    it "is invalid without name" do
+      data = success_data.dup
+      data.delete(:name)
+      slide = Slide.new(data)
+      expect(slide.valid?).to eq(false)
+    end
+
+    it "is invalid without description" do
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->